src/protocols/BUILD.gn

```diff
@@ -21,8 +21,6 @@ static_library("protocols") {
     "echo/Echo.h",
     "echo/EchoClient.cpp",
     "echo/EchoServer.cpp",
```

Comment:
Drive-by: we should take this out for now... Do we still use this? It's not spec compliant. This should only appear in examples/

Reply:
Probably separate PR. I had the same thought.